optimize/client/src/modules/translation/translation.tsx

```diff
@@ -15,7 +15,7 @@ import {loadDateTranslation} from 'dates';
 import {loadTranslation} from './service';
 
 interface TranslationObject {
-  [key: string]: string | TranslationObject;
+  [key: string]: string | number | TranslationObject;
```

Comment:
because of some places where we pass numbers as parameters to be put into the translation string.
For example:
- `optimize/client/src/modules/components/Table/Table.tsx`
- `optimize/client/src/modules/components/KpiCreationModal/KpiCreationModal.tsx`

We can either force using `toString` everywhere or we can allow passing any primitive type so it gets stringified properly in the `translation` module. I will go for the second option but will make it more descriptive